### PR TITLE
[Docs] Fix minor typos in the ownership manifesto.

### PR DESCRIPTION
**Pull request by ravikandhadai**
Fixed minor typos I came across while reading this manifesto.

### CODE REVIEW

File: docs/OwnershipManifesto.md [hunk 1]
```diff
@@ -1381,7 +1381,7 @@ comes from two main sources:
 Otherwise, the feature itself is pretty small.  The compiler
 implicitly emits moves instead of copies, just like we discussed
 above for the `move` intrinsic, and then diagnoses anything
-that that didn't work for.
+that didn't work with the `move` semantics.
```
Comment (CodaFi) [inline docs/OwnershipManifesto.md]: You can probably drop this part of the sentence entirely.  DI's involvement in enforcement is mentioned above already.

**Review by rjmccall — CHANGES_REQUESTED** [review]
The three `that that`s are not errors.  I appreciate the other two fixes, though.

**Comment by ravikandhadai** [comment]
@rjmccall  I rolled back the changes to `that that`s.

**Review by rjmccall — APPROVED** [review]
Thanks, LGTM.

**Comment by ravikandhadai** [comment]
@swift-ci please smoke test and merge

**Pull request merged**